src/app/hearing-part/components/listing-note-list/listing-note-list.component.ts

```diff
@@ -1,73 +1,68 @@
-import { ListingCreateNotesConfiguration } from './../../models/listing-create-notes-configuration.model';
+import { ListingCreateNotesConfiguration } from '../../models/listing-create-notes-configuration.model';
 import { Component, Input, ViewChild } from '@angular/core';
 import { NoteListComponent } from '../../../notes/components/notes-list/note-list.component';
-import { Note } from '../../../notes/models/note.model';
-import { NoteViewmodel, getNoteViewModel } from '../../../notes/models/note.viewmodel';
+import { isOfTypeOtherOrListing, Note } from '../../../notes/models/note.model';
+import { getNoteViewModel, NoteViewmodel, sortNotesByLatestFirst } from '../../../notes/models/note.viewmodel';
```

Comment:
I know it's very minor and it was done automatically by IDE, but I'd suggest to import class first and static methods later.